packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -128,6 +132,24 @@ class FlatTableState<T> extends State<FlatTable<T>>
         !collectionEquals(widget.data, oldWidget.data)) {
       _initData();
     }
+
```

Comment:
can you try to remove this, and add this `addAutoDisposeListener` block back to _TableState, where the diff shows it being deleted? Now that we are computing the `flatListIndex` for the tree table's selected node, and setting `scrollIntoView: true`, the scrolling might work out of the box without having to make any changes to the table code.